backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -849,8 +849,7 @@ template <typename Torus> struct int_are_all_block_true_buffer {
   COMPARISON_TYPE op;
   int_radix_params params;
 
-  int_radix_lut<Torus> *is_max_value_lut;
-  int_radix_lut<Torus> *is_equal_to_num_blocks_lut;
+  std::unordered_map<int, int_radix_lut<Torus> *> is_equal_to_num_blocks_map;
```

Comment:
Name: `is_equal_to_lut_map`
Could you add a comment above saying that this map contains two int_radix_luts, one to compare equality with max_value (the maximum message value) and one to compare equality with num_blocks.